web/global/theme.scss

```diff
@@ -1,8 +1,10 @@
 /* You can add global styles to this file, and also import other style files */
 
 @import '~@angular/material/prebuilt-themes/deeppurple-amber.css';
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500');
```

Comment:
Is this recommended in comparison to having a link in the `index.html`?

Reply:
There seems to be a difference in the way a link-element loads the font, because @import will block parallel downloads ([source](https://stackoverflow.com/a/7199377/1323800)). I'll see if I can change it up and load it from index.html.